public/assets/js/vendor/k.js

```diff
@@ -1,105 +1,149 @@
-		konami.code = callback;
-		konami.load();
-	}
-
-	return konami;
-};
+/*
+ * Konami-JS ~ 
+ * :: Now with support for touch events and multiple instances for 
+ * :: those situations that call for multiple easter eggs!
+ * Code: http://konami-js.googlecode.com/
+ * Examples: http://www.snaptortoise.com/konami-js
+ * Copyright (c) 2009 George Mandis (georgemandis.com, snaptortoise.com)
+ * Version: 1.4.2 (9/2/2013)
+ * Licensed under the MIT License (http://opensource.org/licenses/MIT)
+ * Tested in: Safari 4+, Google Chrome 4+, Firefox 3+, IE7+, Mobile Safari 2.2.1 and Dolphin Browser
+ */
+
+var Konami = function (callback) {
+	var konami = {
+		addEvent: function (obj, type, fn, ref_obj) {
+			if (obj.addEventListener)
+				obj.addEventListener(type, fn, false);
+			else if (obj.attachEvent) {
+				// IE
+				obj["e" + type + fn] = fn;
+				obj[type + fn] = function () {
+					obj["e" + type + fn](window.event, ref_obj);
+				}
+				obj.attachEvent("on" + type, obj[type + fn]);
+			}
+		},
+		input: "",
+        pattern: "38384040373937396665",
+        //bonobo: "667978796679",
+        //memes: "68657875",
+        //roham: "7465666573846968",
+        //doomsday: "657676657285136575666582",
+		load: function (link) {
+			this.addEvent(document, "keydown", function (e, ref_obj) {
+				if (ref_obj) konami = ref_obj; // IE
+				konami.input += e ? e.keyCode : event.keyCode;
+				/*if (konami.input.length > konami.pattern.length)
+					konami.input = konami.input.substr((konami.input.length - konami.pattern.length));*/
+				if (konami.input == konami.pattern) {
+					konami.code(link);
+					konami.input = "";
+                    e.preventDefault();
+					return false;
+				}
+                /*else if (konami.input == konami.bonobo) {
+					konami.code2(link);
+					konami.input = "";
+                    e.preventDefault();
+					return false;
+				}
+                else if (konami.input == konami.roham) {
+                    konami.code3(link);
+                    konami.input = "";
+                    e.preventDefault();
+                    return false;
+                }
+                else if (konami.input == konami.memes) {
+                    konami.code4(link);
+                    konami.input = "";
+                    e.prevenDefault();
+                    return false;
+                }
+                else if (konami.input == konami.doomsday) {
+                    konami.codeD(link);
+                    konami.input = "";
+                    e.preventDefault();
+                    return false;
+                }*/
+			}, this);
+			//this.iphone.load(link);
+		},
+		code: function (link) {
+			window.location = link
+		},
+        code2: function (link) {
+            window.location = link
+        },
+        code3: function (link) {
+            window.location = link
+        },
+        code4: function (link) {
+            window.location = link
+        },
+        /*codeD: function (link) {

```

Comment:
Can you explain what these are? There's no reason to leave commented out code if it's in Git. You can always recover them from the commit history if needed.